subset.py

```diff
@@ -174,6 +160,7 @@ def subset_downstream(self, table, relationships):
             fk_columns = r['fk_columns']
 
             q='SELECT {} FROM {} WHERE {} NOT IN (SELECT {} FROM {})'.format(columns_joined(fk_columns), fully_qualified_table(mysql_db_name_hack(fk_table, self.__destination_conn)), columns_tupled(fk_columns), columns_joined(pk_columns), fully_qualified_table(mysql_db_name_hack(table, self.__destination_conn)))
+            q += " LIMIT {}".format(config_reader.get_max_rows_per_table())
```

Comment:
While the LIMIT feature is great, it doesn't make sense to run a LIMIT on a downstream. This is because every single row in a downstream pass is necessary in order to not break referential integrity. While you can artificially limit data on the direct and upstream pass (because we're optionally grabbing data to build out the subset initially), it doesn't make sense to run a LIMIT here without breaking the intent of condenser.